src/main/java/org/folio/circulation/domain/validation/LoanValidators.java

```diff
@@ -0,0 +1,19 @@
+package org.folio.circulation.domain.validation;
+
+import static org.folio.circulation.support.Result.succeeded;
+import static org.folio.circulation.support.ValidationErrorFailure.singleValidationError;
+
+import org.folio.circulation.domain.Loan;
+import org.folio.circulation.support.Result;
+
+public class LoanValidators {
+
+  private LoanValidators() {}
+
+  public static Result<Loan> refuseWhenLoanIsClosed(Result<Loan> loanResult) {
```

Comment:
How about using this method in DeclareLostResource.class and removing the method duplication?

Reply:
Sure, it has been applied.